server/RecommendationSystem/RecommendationSystem.py

```diff
@@ -0,0 +1,254 @@
+import zipfile
+import dotenv
+import json
+import copy
+import os
+
+import gdown
+import lancedb
+from premai import Prem
+from transformers import AutoModel
+from mongo import db
+
+class RecommendationSystem:
+    dotenv.load_dotenv("../.env")
+
+    model_id = os.environ.get("HF_MODEL_ID")
+    prem_api_key = os.environ.get("PREM_API_KEY")
+    prem_project_id = os.environ.get("PREM_PROJECT_ID")
+
+    def __init__(
+        self,
+        rerank_config,
+    ):
+        device = rerank_config["device"]
+        self.rerank_config = rerank_config
+        self.client = Prem(api_key=self.prem_api_key)
+
+        url = "https://drive.google.com/uc?id=1HpM0rWT9SZr8MTeJ_KMG9G9utWDVn0Qk"
+        output = "./data.zip"
+        data_output_path = "."
+        data_path = ".\data\movies-data"
```

Comment:
Do not hardcode the paths, some are compatible with windows and some for linux. Better store all the paths and everything inside a config, or use `os.path` functionalities or `pathlib` to get absolute paths.